packages/venia-concept/src/RootComponents/Category/container.js

```diff
@@ -0,0 +1,18 @@
+import { connect } from 'src/drivers';
+
+import Category from './category';
+import { setCurrentPage, setPrevPageTotal } from 'src/actions/catalog';
+
+const mapStateToProps = ({ catalog }) => {
+    return {
+        currentPage: catalog.currentPage,
+        pageSize: catalog.pageSize,
+        prevPageTotal: catalog.prevPageTotal
+    };
+};
+const mapDispatchToProps = { setCurrentPage, setPrevPageTotal };
+
+export default connect(
+    mapStateToProps,
+    mapDispatchToProps
+)(Category);
```

Comment:
I think this entire connection to redux could be removed. I'll investigate a bit more.